package.json

```diff
@@ -6,15 +6,15 @@
   "description": "Home Assistant websocket client",
   "source": "lib/index.ts",
   "types": "dist/index.d.ts",
-  "main": "dist/haws.umd.js",
+  "main": "dist/haws.umd.cjs",
```

Comment:
So this is a breaking change for anyone that currently relies on the UMD build. I think that @bramkragten had another idea with a second `package.json`. 

Alternatively, we can also adopt a support strategy where we always support the latest LTS and so drop anything Node 10-ish.